### PR TITLE
Improving instrumentation.

### PR DESCRIPTION
**Pull request by enxebre**
Some changes for sticking to best practices on instrumentation according to comments here https://github.com/containous/traefik/pull/1022 ping @brian-brazil @SantoDE

### CODE REVIEW

File: docs/toml.md [hunk 1]
```diff
@@ -718,7 +722,7 @@ $ curl -s "http://localhost:8080/api" | jq .
 - `/metrics`: You can enable Traefik to export internal metrics to different monitoring systems (Only Prometheus is supported at the moment).
 
 ```bash
-$ traefik --web.metrics.prometheus --web.metrics.prometheus.buckets="100,300"
+$ traefik --web.metrics.prometheus --web.metrics.prometheus.buckets="0.1,0.3"
```
Comment (brian-brazil) [inline docs/toml.md]: I'd keep this in line with the defaults.

Histograms only work when all the servers you're aggregating across have the same buckets, so by having this consistent it reduces the scope for mishaps.

Comment (SantoDE) [inline docs/toml.md]: Agreed, that'd be good

File: docs/toml.md [hunk 2]
```diff
@@ -544,6 +544,10 @@ address = ":8080"
 # [web.statistics]
 #   RecentErrors = 10
 #
+# To enable Traefik to export internal metrics to Prometheus
+# [web.metrics.prometheus]
+#   Buckets=[0.1,0.5]
```
Comment (SantoDE) [inline docs/toml.md]: Please adjust the docs to match with the defaults as well :-)

File: traefik.sample.toml [hunk 3]
```diff
@@ -326,6 +326,11 @@
 # Enable more detailed statistics
 # [web.statistics]
 #   RecentErrors = 10
+#
+# To enable Traefik to export internal metrics to Prometheus
+# [web.metrics.prometheus]
+#   Buckets=[0.1,0.5]
```
Comment (SantoDE) [inline traefik.sample.toml]: Same here ;-)

**Comment by enxebre** [comment]
Should be ok now

**Comment by SantoDE** [comment]
Cool @enxebre 

LGTM now 👼 

Would you be so kind to just rebase the branch and squash your commits? 

ping @containous/traefik

**Comment by enxebre** [comment]
Hey @SantoDE I just rebased it, just in case you didn't know, you can squash and merge the PR from Github straight away  
screen shot 2017-01-17 at 17 18 47

**Pull request merged**